### DISCUSSION TITLE
Mod initialization

### DISCUSSION THREAD

**Discussion by SkyTrax79** [ramensoftware/windhawk-mods#1064] [Q&A]
Hi, I have a couple of mods for the Windows 11 interface and the one that is Windows 11 Taskbar Styler is not initializing automatically on Windows startup, I have to toggle the On/Off witch manually each time to reenable it.  The switch in the On position on startup, just that the mod is not active.  It's the only mod that's doing this, the others are fine.  Is there a setting that I missed ?

Thanks a lot

**Comment by m417z** [comment]
Are you using other customization apps such as TranslucentTB? Some customization tools, like the mod, use XAML diagnostics to monitor for taskbar changes, and there can only be one active monitoring process at a time.

**Comment by SkyTrax79** [comment]
Ah ok, yes I'm using TranslucentTB and Explorer patcher on top of that only to hide the Recommended section of the Start menu, I keep the Windows 11 taskbar style but with the mod Taskbar Labels to make the buttons all the same width.  It's not a big deal anyway, when I activate the taskbar styler mod after it has bootied it will stick well with the others mods.



Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by m417z** [reply]
> hide the Recommended section of the Start menu

You can also use [Windows 11 Start Menu Styler](https://windhawk.net/mods/windows-11-start-menu-styler) for that.

Also, Windows 11 Taskbar Styler is quite powerful, perhaps you can replace TranslucentTB and have this issue fixed. There's also the new [Taskbar Background Helper](https://windhawk.net/mods/taskbar-background-helper) mod.

**Comment by SkyTrax79** [comment]
Ok thanks for the reply, I will try and experiment with this.


Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by SkyTrax79** [comment]
Hi, you were right about the initialization mods, I solved the problem by setting the Windhawk service to Delayed Start mode.

You helped me a while ago to find the targets to change the fonts in places like the Start Menu and I managed to find almost all of them, I did a Windows repair install 2 days ago and I don't know why but in some places the modified fonts don't work anymore, in the Start Menu and the fly out menu when you right click on an app taskbar button, the rest works.  I reinstalled Windhawk on top without success, I used the utility to download the symbols but that doesn't work either.  In the default utility, Undecorated is checked, but there's also Decorated just beside, which I'd also checked. What's the difference between the 2, and once the symbols have been downloaded, do I need to do anything else?  I'm not even sure if it's a problem with the symbols, do you have any other ideas for solving the issue?

Thanks in advance

________________________________
De : Michael Maltsev ***@***.***>
Envoyé : 11 octobre 2024 17:58
À : ramensoftware/windhawk-mods ***@***.***>
Cc : SkyTrax79 ***@***.***>; Author ***@***.***>
Objet : Re: [ramensoftware/windhawk-mods] Mod initialization (Discussion #1064)


Are you using other customization apps such as TranslucentTB? Some customization tools, like the mod, use XAML diagnostics to monitor for taskbar changes, and there can only be one active monitoring process at a time.

**Reply by m417z** [reply]
> but in some places the modified fonts don't work anymore

If only some customizations don't work, then you're probably not using the right targets, or not enough of them. The best place to ask about it is here:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide/issues

**Comment by SkyTrax79** [comment]
That's strange because nothing has changed for the list of targets, like the first one in the Start Menu Styler is TextBlock[2] with FontFamily="custom font", that was the first one I made that is for changing the font's name of the pinned apps., in fact all the fonts of the Start Menu don't work and there is many with some that have a more meaningful name like TextBlock#AllAppsHeading that changes only the word "All >" in the top right corner, the target and Font family are there.  Strangely the following targets that serve for the background drop work.


________________________________
De : Michael Maltsev ***@***.***>
Envoyé : 6 mars 2025 16:03
À : ramensoftware/windhawk-mods ***@***.***>
Cc : SkyTrax79 ***@***.***>; Author ***@***.***>
Objet : Re: [ramensoftware/windhawk-mods] Mod initialization (Discussion #1064)


but in some places the modified fonts don't work anymore

If only some customizations don't work, then you're probably not using the right targets, or not enough of them. The best place to ask about it is here:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide/issues

**Comment by SkyTrax79** [comment]
Ok, It's fixed, I just reinstalled the custom font.  That doesn't explain why it was working only at some places because I only use one custom font, anyway no big deal.

Thanks



________________________________
De : Michael Maltsev ***@***.***>
Envoyé : 6 mars 2025 16:03
À : ramensoftware/windhawk-mods ***@***.***>
Cc : SkyTrax79 ***@***.***>; Author ***@***.***>
Objet : Re: [ramensoftware/windhawk-mods] Mod initialization (Discussion #1064)


but in some places the modified fonts don't work anymore

If only some customizations don't work, then you're probably not using the right targets, or not enough of them. The best place to ask about it is here:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide/issues
